docs/development/issue_labeling.md

```diff
@@ -0,0 +1,228 @@
+# Issue labeling
+
+We try to keep issues well-classified by labeling them appropriately.
+Only members/collaborators with the required rights can apply labels.
+
+## Basic knowledge about Renovate
+
+You should know about platforms, package managers, datasources and versioning to label issues effectively.
+
+- To learn about platforms, read the [Renovate docs on Platforms](https://docs.renovatebot.com/modules/platform/).
+- To learn about managers, read the [Renovate docs on Managers](https://docs.renovatebot.com/modules/manager/).
+- To learn about datasources, read the [Renovate docs on Datasources](https://docs.renovatebot.com/modules/datasource/).
+- To learn more about versioning, read the [Renovate docs on Versioning](https://docs.renovatebot.com/modules/versioning/).
+
+## Label categories
+
+### Severity
+
+<details>
+    <summary>Severity</summary>
+
+    breaking
+    major
+
+</details>
+
+QUESTION: When/who should use these severity labels?
+
+### Priority
+
+<details>
+    <summary>Priority</summary>
+
+    pri0-blocker
+    pri1-critical
+    pri2-important
+    pri3-normal
+    pri4-low
+    wontfix
+
+</details>
+
+Use these to assign a priority level to an issue.
+
+QUESTION: Who gets to decide the priority level? @rarkins and maintainers, or can a triaging member select a priority level?
```

Comment:
Any collaborator in the repo with rights can assign any label

Reply:
I know that any collaborator with the required rights can assign labels, the question I'm really asking here is: "Who decides what priority an issue is? Is a best-effort guess from any collaborator okay?"

I hope I'm making sense here... :smile:

Reply:
Yes, that is ok. Nothing bad happens if someone has a different opinion on priority.